tests/core/test_metrics_calculator.py

```diff
@@ -125,3 +127,32 @@ def test_column_partition_metric(
         )
         for idx, element in enumerate(results[desired_metric.id])
     )
+
+
+@mock.patch("great_expectations.execution_engine.execution_engine.ExecutionEngine")
+def test_get_metric_calls_get_metrics_and_returns_correct_result(
+    execution_engine: mock.MagicMock,
+):
+    """
+    The purpose of this basic test is to insure that:
+        1) MetricsCalculator.get_metric() calls MetricsCalculator.get_metrics() exactly once; and
+        2) MetricsCalculator.get_metric() correctly retrieves result from dictionary, returned by
+           MetricsCalculator.get_metrics() by using the "metric_name" as the key.
+    """
+    metrics_calculator = MetricsCalculator(execution_engine=execution_engine)
+    metric_name = "my_metric_name"
+    metric_value = "my_metric_value"
+    with mock.patch(
+        "great_expectations.validator.metrics_calculator.MetricsCalculator.get_metrics",
+        return_value={metric_name: metric_value},
```

Comment:
```suggestion
        return_value={metric_name: metric_value, "wrong_metric": "wrong_value"},
```

Reply:
@Kilo59 What is the reason for name changes?  As far as I understand, we are testing for "actual" (not "wrong") values.  Or are you recommending an additional test?  Happy to discuss.   Thanks.